lnrpc/gen_protos_docker.sh

```diff
@@ -6,7 +6,7 @@ set -e
 DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
 
 # golang docker image version used in this script.
-GO_IMAGE=golang:1.20.4-alpine
+GO_IMAGE=docker.io/library/golang:1.20.4-alpine
```

Comment:
Looks good. Though it's not 100% clear to me why this change isn't also needed in the `lnrpc/Dockerfile`? Is it that podman automatically falls back to this URI within a `Dockerfile` (since the name implies Docker)?